sn_node/src/bin/faucet/faucet_server.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use std::path;
+
+use crate::{claim_genesis, send_tokens};
+use eyre::{eyre, Result};
+use sn_client::Client;
+
+use tiny_http::{Response, Server};
+
+/// Run the faucet server.
+///
+/// This will listen on port 8000 and send tokens to any request.
+///
+/// # Example
+///
+/// ```bash
+/// # run faucet server
+/// cargo run  --features="local-discovery" --bin faucet --release -- server
+///
+/// # query faucet server for DBC at `get local wallet address`
+/// curl "localhost:8000/`cargo run  --features="local-discovery"  --bin safe --release  wallet address | tail -n 1`" > dbc_hex
+///
+/// # feed DBC to local wallet
+/// cat dbc_hex | cargo run  --features="local-discovery"  --bin safe --release  wallet read
+///
+/// # balance should be updated
+/// ```
```

Comment:
Docs are here for now, might be useful to have them where we need them

Reply:
will this work w/o local-discovery?

Reply:
I tried with local-discovery and got this error
```
willie@gagarin:~/projects/maidsafe/safe_network$ cargo run  --features="local-discovery" --bin faucet --release -- server
    Finished release [optimized] target(s) in 0.25s
     Running `target/release/faucet server`
🔗 Connected to the Network                                                                                                                                thread 'main' panicked at 'Faucet wallet shall be created successfully.: Bincode(Custom("deserialized bytes don't encode a group element"))', /home/willie/projects/maidsafe/safe_network/sn_transfers/src/dbc_genesis.rs:108:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```